bigtable/client/client_options_test.cc

```diff
@@ -18,25 +18,56 @@
 
 TEST(ClientOptionsTest, ClientOptionsDefaultSettings) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
-  EXPECT_EQ("bigtable.googleapis.com", client_options_object.endpoint());
+  EXPECT_EQ("bigtable.googleapis.com", client_options_object.data_endpoint());
+  EXPECT_EQ("bigtableadmin.googleapis.com",
+            client_options_object.admin_endpoint());
   EXPECT_EQ(typeid(grpc::GoogleDefaultCredentials()),
             typeid(client_options_object.credentials()));
 }
 
-TEST(ClientOptionsTest, ClientOptionsCustomEndpoint) {
-  setenv("BIGTABLE_EMULATOR_HOST", "testendpoint.googleapis.com", 1);
+namespace {
+class ClientOptionsEmulatorTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    // TODO(#100) - setenv() is a Unix specific call.
+    previous_ = std::getenv("BIGTABLE_EMULATOR_HOST");
+    setenv("BIGTABLE_EMULATOR_HOST", "testendpoint.googleapis.com", 1);
+  }
+  void TearDown() override {
+    if (previous_) {
+      setenv("BIGTABLE_EMULATOR_HOST", previous_, 1);
+    } else {
+      unsetenv("BIGTABLE_EMULATOR_HOST");
+    }
+  }
+
+ protected:
+  char const *previous_ = nullptr;
+};
+}  // anonymous namespace
+
+TEST_F(ClientOptionsEmulatorTest, Simple) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
-  EXPECT_EQ("testendpoint.googleapis.com", client_options_object.endpoint());
+  EXPECT_EQ("testendpoint.googleapis.com",
+            client_options_object.data_endpoint());
+  EXPECT_EQ("testendpoint.googleapis.com",
+            client_options_object.admin_endpoint());
   EXPECT_EQ(typeid(grpc::InsecureChannelCredentials()),
             typeid(client_options_object.credentials()));
-  unsetenv("BIGTABLE_EMULATOR_HOST");
 }
 
 TEST(ClientOptionsTest, EditEndpoint) {
```

Comment:
s/EditEndpoint/EditDataEndpoint/ for clarity.

Reply:
Done.